src/clinical/donor-repo.ts

```diff
@@ -392,7 +393,17 @@ export const donorDao: DonorRepository = {
       submitterId: { $in: submitterIds },
       programId: programId,
     });
-    const mapped = result.map((d: DonorDocument) => {
+    const mapped = result.map(d => {
+      return MongooseUtils.toPojo(d) as Donor;
```

Comment:
you, using `as`? le wut!

Reply:
Old mongoose code, there are better TS ways to do this now. Following convention of the file instead of investing time in a re-write of the DB management.

Reply:
screenshotted to use without context in the future (context provider is undefineed) :p